src/Map/TileMap.cpp

```diff
@@ -607,7 +607,7 @@ Tile* TileMap::getVisibleTile(int x, int y, int level)
 
 bool TileMap::isVisibleTile(int _x, int _y, int _d) const
 {
-	if (!isPointInRect(_x, _y, mMapViewLocation.x(), mMapViewLocation.y(), mEdgeLength - 1, mEdgeLength - 1))
+	if (!NAS2D::Rectangle{mMapViewLocation.x(), mMapViewLocation.y(), mEdgeLength - 1, mEdgeLength - 1}.contains(NAS2D::Point{_x, _y}))
```

Comment:
Consider making this a variable for readability. I don't know what the rectangle represents.

Reply:
Neither do I, which is one reason why I left it. The original code doesn't give any indication as to what those coordinates represent.

Reply:
Looking at it more closely, `mMapViewLocation` is probably a topleft camera position and mEdgeLength is probably extents of said view.

Reply:
That said, since it's culling anything outside that area: `cullBounds` is appropriate

Reply:
In world space of course (i.e., tile coordinates)